### ISSUE TITLE
DX11 is missing in the renderers list

### ISSUE THREAD

**Issue by Nasher73** [TVTower/TVTower#639]
There is a TVTower_DirectX11.bat in the game's root folder but you can't select DirectX11 from the renderers drop list.

**Comment by GWRon** [comment]
Yes, with the current master branch (and a compatible Windows OS) you get DX11 in the dropdown.

**Comment by GWRon** [comment]
Addressed in f96f05cb35d8f8d2bf7ac337c1c44189b918f450
